Cargo.toml

```diff
@@ -362,7 +362,7 @@ rand_xorshift = "0.3"
 rayon = "1.10"
 redis = "0.23.0"
 reed-solomon-erasure = { version = "6.0.0", features = ["simd-accel"] }
-regex = "1.7.1"
+regex = "1.10.6"
```

Comment:
There's a 1.11 version out there. Maybe we can just have regex version "1" so that it automatically picks the latest?

Reply:
You shouldn't need to modify `Cargo.toml` for this. Just `cargo update -p regex` should be sufficient to get it to reevaluate the dependencies it selected. It will be able to pick anything between `1.7.1` and `2.0.0`. Having bounds as wide as possible is arguably a good thing as it gives the dependency resolver more flexibility to minimize the number of duplicates.

Reply:
good to know, thanks @nagisa, I'm reverting regex version bump then